diff -ura dmenu-4.8.orig/config.h dmenu-4.8.new/config.h
--- dmenu-4.8.orig/config.h	2018-04-08 09:19:33.981174770 -0700
+++ dmenu-4.8.new/config.h	2018-04-08 09:16:16.569178306 -0700
@@ -4,13 +4,13 @@
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"monospace:size=16"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
+	[SchemeNorm] = { "#00FF00", "#000000" },
+	[SchemeSel] = { "#000000", "#00FF00" },
 	[SchemeOut] = { "#000000", "#00ffff" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
diff -ura dmenu-4.8.orig/config.mk dmenu-4.8.new/config.mk
--- dmenu-4.8.orig/config.mk	2018-03-14 12:26:40.000000000 -0700
+++ dmenu-4.8.new/config.mk	2018-04-08 09:10:04.569184967 -0700
@@ -29,3 +29,6 @@
 
 # compiler and linker
 CC = cc
+CPPFLAGS+=-D_FORTIFY_SOURCE=2
+CFLAGS+=-march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt
+LDFLAGS+=-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now
