diff -ura dmenu-4.8.orig/config.def.h dmenu-4.8.new/config.def.h
--- dmenu-4.8.orig/config.def.h	2018-04-08 09:19:33.981174770 -0700
+++ dmenu-4.8.new/config.def.h	2018-04-08 09:16:16.569178306 -0700
@@ -4,13 +4,13 @@
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"monospace:size=12"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
+	[SchemeNorm] = { "#00FF00", "#000000" },
+	[SchemeSel] = { "#000000", "#00FF00" },
 	[SchemeOut] = { "#000000", "#00ffff" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
